module/epochs/qc_voter.go

```diff
@@ -58,6 +61,7 @@ func NewRootQCVoter(
 		state:             state,
 		qcContractClients: contractClients,
 		wait:              time.Second * 10,
+		unit:              engine.NewUnit(),
```

Comment:
```suggestion
		mu:              sync.Mutex{},
```
We're only using the lock part of `Unit`, better to just use a lock directly